charger/easee.go

```diff
@@ -363,6 +364,20 @@ func (c *Easee) ProductUpdate(i json.RawMessage) {
 		// startup completed
 		c.once.Do(func() { close(c.done) })
 
+	case easee.ERROR_CODE:
+		newErrorCode := value.(int)
+
+		// ERROR_CODE 2048:  "Overcurrent", Easee App "Überstrom erkannt" / red  LED stripe, OpMode 5
+	 	// [easee ] TRACE ... ProductUpdate : ... ERROR_CODE 2048
+       	// [easee ] TRACE ... ProductUpdate : ... ERROR_STRING [ERROR] Charger: Overcurrent EV=18637 > 17000 mA
+		// Reboot or plug out/in solves this issue.
+    	if c.errorCode != 2048 && newErrorCode == 2048 {
```

Comment:
The errorCode ProductUpdate is very sticky. 
Recent example from my charger, reporting an ERROR_CODE value from October 2023, i.e. 5 months old:

```
[easee ] TRACE 2024/02/09 02:08:40 ProductUpdate XXXXXXXX: (2023-10-23 10:00:38 +0000 UTC) ERROR_CODE 0
```

Have you double checked the behavior of this value in such an error case? If the charger does not reset this, but instead keeps repeating an old ProductUpdate, this code would reboot the charger on every evcc startup.